docs/blog/2018-06-15-why-we-built-the-site-showcase/index.md

```diff
@@ -15,76 +15,76 @@ We recently finished the first phase of the [Site Showcase](https://next.gatsbyj
+We built the showcase to make it easier for you to find the src code for open source Gatsby sites and easier to show off what Gatsby can do to potential clients / coworkers.
 
-## Source code
+### Src code
 
-Many people learn how to build things by looking at src code for other sites. As you browse through the site showcase, look for the GitHub icon on each site’s detail page which links you to the source code (not all sites will have it):
+Sometimes, examples are worth 1000 words. That's why we added a GitHub icon next to sites with open src code.
 
-![Site Showcase Source Code Available Icon](site-showcase-source-code.png)
+![Site Showcase Src Code Available Icon](site-showcase-source-code.png)
 
-## Showing off what Gatsby can do
+### Showing off what Gatsby can do
 
 The site showcase can help you pitch Gatsby to clients and coworkers as the best option for their next project. As they search the site showcase, it will provide a vision of what developers can accomplish with Gatsby.
 
-# UX research process
+## UX research process
 
 Here’s where we discussed and built the site showcase if you'd like to go through the discussion that led to this project:
 * [Site Showcase Issue](https://github.com/gatsbyjs/gatsby/issues/4392)
 * [Site Showcase WIP PR](https://github.com/gatsbyjs/gatsby/pull/5524)
 
-## Empathy map
+### Empathy map
 
 Here’s a summary of what I learned from over 80 interviews with Gatsby users. This "empathy map" describes what kinds of pain points we're trying to solve with the site showcase.
 
 ![Empathy Map Site Showcase](empathy-map-site-showcase.jpg)
  
-## User Flow
+### User Flow
 
 The empathy map reflects how it took most people a lot of time browsing through the Awesome Gatsby list to find example sites and how many people share their sites through Twitter and there isn’t enough time or room for everyone to get retweeted by the @Gatsbyjs account.
 
 With the empathy map and the interviews as our guide, we created a list of jobs to be done and imagined the typical flow a user would take through the site showcase. We did this to make sure we planned both a sensible information architecture and the necessary functionality to meet user’s main needs.
 
 ![User flow](user-flow-site-showcase.jpg)
  
-## Defining the MVP
+### Defining the MVP
 
-Here are the features that clearly needed to be included in the MVP; you'll notice that we checked all the boxes thanks to the hard work of many people!
+We thought the following features were important to include; you'll notice that the boxes are all checked. This is all possible because Cassie (@ThatOtherPerson) and Shawn (@tsiq-swyx) put in a huge effort to get this developed, and Flo (@fk) did an amazing job on design. Many thanks to other people who contributed, including @gillkyle, @LeKoArts, @davidluhr, @daydream05, @m-allanson, @pieh, @calcsam and more (these are all GitHub names).
```

Comment:
I think @sw-yx is Shawn's GitHub username: https://github.com/sw-yx

Reply:
A link to each of these would be great, too.

Reply:
yes use my main github username please

Reply:
ok thanks for clarification!